starter_code/css/relaxr2.css

```diff
@@ -0,0 +1,185 @@
+html, body {
+    padding: 0;
+    margin: 0;
+    font-family: 'Open Sans', serif;
+    background-color: #f0efef;
+}
+
+.header {
+    background-color: #033048;
+    padding: 20px 20px;
+}
+
+.header ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.header ul li a {
+    color: #f9e42e;
+    text-decoration: none;
+    font-weight: 500;
+    padding-right: 15px;
+}
+
+.header ul li a:hover {
+    opacity: .5;
+}
+
+.header ul li.logo a {
+    color: white;
+    font-size: 40px;
+    font-weight: 800;
+    align-self: flex-start;
+    margin-right: 200px;
+}
+
+.nav {
+    font-size: 24px;  
+    font-weight: 600;
+}
+
+img:hover {
+    opacity: 0.3;
+}
+
+.left {
+    width: 700px;
+    padding-top: 30px;
+    padding-left: 55px;
+    padding-bottom: 100px;
+}
+
+.left h2 {
+    font-size: 40px;
+    font-weight: 600;
+    color:#033048;
+    margin-top: 30px;
+}
+
+.left h2:hover{
+    opacity: .5;
+}
+
+.left p {
+    font-size: 24px;
+    font-weight: 300;
+    initial-letter: 3;
+    color: #606161;
+}
+
+p::first-letter {
+    font-size: 200%;
+}
+
+h3 {
+    font-size: 24px;
+    font-weight: 600;
+    text-align: right;
+    color: #033048;
+    margin-top: 40px;
+}
+
+h3:hover {
+    opacity: .5;
+}
+
+.right {
+    width: 240px;
+    color: #033048;
+    float: right;
```

Comment:
@srhjang I would have used Flexbox here to line up al the children elements to the right.